packages/scipy/patches/skip-fortran-fails-to-link.patch

```diff
@@ -0,0 +1,125 @@
+commit c3affe05b5cb8d992cb7a955ff8b4c79452f3cc9
+Author: Roman Yurchak <rth.yurchak@pm.me>
+Date:   Thu Oct 11 17:10:36 2018 +0200
+
+    Skip fortran code that fails to link
+    
+       scipy.integrate.odepack
```

Comment:
Looks like I have not kept the error message, but this was failing at link time. It's fortran files, so it might have something to do with f2c as well.